joomla/joomla-cms
# Removed Latest Version Check Help

Pull Request for Issue # .

### Summary of Changes

Remove Latest Version Check Help screen from the Joomla! Help page in Joomla 4. 

### Testing Instructions

In Joomla 3 or Joomla 4 go to Help / Joomla! Help. The second item in the index is Latest Version Check. In Joomla 3 there is a warning if the installed version is not the current version and then a list of Announcements and Release notes going back through all major/minor/patch versions of the 3.x series. They link to the Release News pages. There are also Frequently Asked Questions for each major/minor/patch release. They are actually bug reports, are sparce and don't work properly when delivered by the Help proxy server. All of this has accumulated well over 400 Help support files. The Version Check is redundant because the Home dashboard does that check anyway in Joomla 4. 

In Joomla 4 the Latest Version Check Help link returns a 404 not found page because there are no Help support pages (yet).

This PR removes that Help page and the code required to generate it - all in com_admin.

### Actual result BEFORE applying this Pull Request

The Latest Version Check page is present in the Joomla! Help index for Joomla 4 but returns an error page.

### Expected result AFTER applying this Pull Request

The Latest Version Check page is absent from the Joomla! Help index for Joomla 4.

### Documentation Changes Required

None
